pages/Configuring/Animations.md

```diff
@@ -11,27 +12,30 @@ or
 animation=NAME,ONOFF,SPEED,CURVE
 ```
 
-for example:
-
-```ini
-animation=workspaces,1,8,default
-animation=windows,1,10,myepiccurve,slide
-```
-
 `ONOFF` can be either 0 or 1, 0 to disable, 1 to enable.
 
 `SPEED` is the amount of ds (1ds = 100ms) the animation will take
 
 `CURVE` is the bezier curve name, see [curves](#curves).
 
-`STYLE` (optional) is the animation style
+`STYLE` is the animation style
```

Comment:
why? `(optional)` will make it easier for people who just skim over the wiki to notice